#goal definition
MissionPoint current
MissionPoint target
float32 azimuth
---
#result definition
bool target_reached
---
#feedback
float32 way_completed